src/pages/HomePage.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Col, Jumbotron, Row } from 'react-bootstrap';
+
+
+const HomePage = () => (
+  <div>
+    <Row>
+      <Col sm={12} md={8} mdOffset={2} lg={6} lgOffset={3}>
+		   <Jumbotron>
+		    <h1>Welcome!</h1>
+		    <p>UltiManager allows you to record statistics for a new game, or view the results of previous games.</p>
+		    <h2>Please log in to view your teams, games and statistics</h2>
+		    
+		  </Jumbotron>
+		  <img class="img-fluid"style={{width:'100%'}} src="./frisbeePhoto.jpg" alt="Frisbee 1" ></img>
```

Comment:
A few comments on this.

### 1. Images should use bootstrap's `<Image>` component.

This component can be imported from `react-bootstrap`.

### 2. Images should be imported into the component

Instead of `src="./some/path"`, add the following:

```jsx
import image from './images/my-image.jpg';

// And then in your component:
<Image src={image} />
```